install.sh

```
@@ -0,0 +1,1349 @@
+CUSTOMER=""
+VERSION=""
+OFFLINE_IP=""
+DEPLOY_MODE="online"  # "online" 또는 "offline" - determine_deployment_mode에서 설정
+NON_INTERACTIVE=false
+DRY_RUN=false
+RESET=false
+SKIP_ASTRAGO=false
+ASTRAGO_ONLY=false
+# 스크립트 위치 = 프로젝트 루트
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+DEPLOYMENT_ROOT="${SCRIPT_DIR}"  # install.sh가 루트에 있으므로 동일
```

Shellcheck (0.11.0)

[warning] DEPLOYMENT_ROOT appears unused. Verify use (or export if used externally).

(SC2034)